benchmarks/nel/scripts/custom_functions.py

```diff
@@ -1,24 +1,65 @@
 """ Custom functions to be hooked up into the registry. """
 from functools import partial
-
-import spacy
 from typing import Iterable, Callable
-
 import typing
+import spacy
 from spacy.kb import Candidate, KnowledgeBase
 from spacy.tokens import Span
-from scripts.candidate_generation.embeddings import create_candidates
+
+# More elegant way to resolve import conflicts between training and evaluation calls?
+try:
+    import scripts.candidate_generation.embeddings as embedding_candidate_generation
+    import scripts.candidate_generation.lexical as fuzzy_lexical_candidate_generation
+except ModuleNotFoundError:
+    import candidate_generation.embeddings as embedding_candidate_generation
+    import candidate_generation.lexical as fuzzy_lexical_candidate_generation
+
```

Comment:
I need to look at how other projects do this but I think there is a better way, like you note.

Reply:
Have you found an alternative to this yet?

Reply:
Is there a way to set `PYTHONPATH` in `project.yml`? Running `PYTHONPATH=. python ...` fails, and setting env variables directly in `project.yml` isn't possible (right?). `PYTHONPATH=. spacy project run ...` should do the trick, but I'd rather not require users to set this themselves.

Reply:
There are ways to do it, but they are all kind of annoying, so I'm not sure they're easier than what you've got overall.

> Running PYTHONPATH=. python ... (or with any value for PYTHONPATH) as a project command fails as it's not parsed properly,

Specifying env vars that way is a bashism, which is why it doesn't work. There are different ways to do this but one would be `env PYTHONPATH=. python ...`. This is probably the simplest alternative, but is still kind of weird.

You could also put the code in a shell script, though that would be silly for a single line command. 

> setting env variables directly in project.yml isn't possible (right?).

No, I don't think that's possible.

Reply:
Ah, that's why - thanks! Still kind of weird, but better than the `try`-`except` imports...